tests/clients/digitalOcean/test_digitaloceanclient.py

```diff
@@ -0,0 +1,193 @@
+
+        **.Test Scenario**
+        #. Get all the images names
+        """
+        self.assertIsNotNone(self.dg.get_image_names)
+
+    @pytest.mark.integration
+    def test07_test_get_digital_ocean_images_names(self):
+        """Test for getting digital ocean available images by specific name
+
+        **.Test Scenario**
+        #. Get all the images by specific name
+        """
+        self.assertIsNotNone(self.dg.get_image_names("centos"))
+
+    @pytest.mark.integration
+    def test08_test_get_digital_ocean_sizes(self):
+        """Test for getting digital ocean available sizes
+
+        **.Test Scenario**
+        #. Get all the available sizes
+        """
+        self.assertIsNotNone(self.dg.sizes)
+
+    @pytest.mark.integration
+    def test09_test_get_digital_ocean_regions(self):
+        """Test for getting digital ocean available regions
+
+        **.Test Scenario**
+        #. Get all the available regions
+        """
+        self.assertIsNotNone(self.dg.regions)
+
+    @pytest.mark.integration
+    def test10_test_get_digital_ocean_region_names(self):
+        """Test for getting digital ocean available region names
+
+        **.Test Scenario**
+        #. Get all the available region names
+        """
+        self.assertIsNotNone(self.dg.region_names)
+
+    @pytest.mark.integration
+    def test11_test_get_digital_ocean_region(self):
+        """Test for getting digital ocean specific region
+
+        **.Test Scenario**
+        #. Get all the available regions
+        #. Get specific region
+        """
+        regions = self.dg.region_names
+        self.assertIsNotNone(self.dg.get_region(regions[0]))
+
+    @pytest.mark.integration
+    def test12_test_get_digital_ocean_ssh_keys(self):
+        """Test for getting digital ocean available sshkeys
+
+        **.Test Scenario**
+        #. Get all the sshkeys
+        """
+        self.assertIsNotNone(self.dg.sshkeys)
+
+    @pytest.mark.integration
+    def test13_test_get_digital_ocean_get_default_ssh_key(self):
+        """Test for getting digital ocean default sshkey
+
+        **.Test Scenario**
+        #. Get default sshkey
+        """
+        self.assertIsNotNone(self.dg.get_default_sshkey())
+
+    @pytest.mark.integration
+    def test14_test_get_digital_ocean_get_specific_ssh_key(self):
+        """Test for getting digital ocean specific sshkey
+
+        **.Test Scenario**
+        #. Get all the available sshkeys 
+        #. Get specific sshkey by name
+        """
+        ssh_keys = self.dg.sshkeys
+        self.assertIsNotNone(self.dg.get_sshkey(ssh_keys[0].name))
+
+    def wait_instance_deploy(self, seconds, check_exist_method, name):
+        for _ in range(seconds):
+            if check_exist_method(name):
+                return True
+            time.sleep(1)
+        return False
+
+    def tearDown(self):
+        for item in self.dg_instances:
+            item.delete_remote()
```

Comment:
This will delete local too?

Reply:
Also, I an ssh key and lots of instances for other factories are created in `setUp` without cleanup here.

Reply:
It will delete  what was deployed on the digital ocean not local